arbitrum-client/src/helpers.rs

```diff
@@ -18,3 +22,10 @@ pub fn deserialize_retdata<'de, T: Deserialize<'de>>(
 ) -> Result<T, ArbitrumClientError> {
     postcard::from_bytes(retdata).map_err(|e| ArbitrumClientError::Serde(e.to_string()))
 }
+
+/// Computes the Keccak-256 hash of the serialization of a scalar,
+/// used for filtering events that have indexed scalar topics
+pub fn hash_scalar(scalar: Scalar) -> Result<H256, ArbitrumClientError> {
```

Comment:
I'd propose renaming this to avoid name collisions + confusion elsewhere in the relayer codebase. Maybe something that indicates it's a keccak hash, like `keccak_hash_scalar`?